### DISCUSSION TITLE
Issue with run_ss3sim function

### DISCUSSION THREAD

**Discussion by kinmic** [ss3sim/ss3sim#395] [Q&A] [answered]
Hi there,

I was hoping you could please help me with an issue I'm having. I'm currently trying to work through the example of the ss3sim as outlined in [this vignette](https://ss3sim.github.io/ss3sim/articles/introduction.html#setup-self-test). All the df set up works fine but when I attempt to run the simulation I get this error 
"Error in ss3sim_base(agecomp_params = list(cpar = list(sa.cpar.1 = NA_integer_,  : 
  unused argument (`NA` = TRUE)"

Any ideas what might be causing this and how I can rectify it?

I'm new to this package and would really appreciate any help you could offer this. Also as a sidenote I have installed both r4ss and ss3sim from the github repo and not CRAN.

# Code:
library(r4ss)
library(ss3sim)

df <- setup_scenarios_defaults(nscenarios = 2)
df[, "bias_adjust"] <- FALSE
df[, "hess_always"] <- TRUE
df[, grep("^cf\\.", colnames(df))]
df[, grep("^s[al]\\.", colnames(df))]
df[, "si.years.2"] <- "seq(76, 100, by = 2)"
df[, "si.sds_obs.2"] <- c(0.1, 0.4)
df <- rbind(df, df)
df[, "ce.par_name"] <- "NatM_uniform_Fem_GP_1"
df[, "ce.par_int"] <- NA
df[, "ce.par_phase"] <- rep(c(-1, 3), each = 2)
df[, "scenarios"] <- c("D0-E0-F0-cod", "D1-E0-F0-cod",
                       "D0-E1-F0-cod", "D1-E1-F0-cod")
iterations <- 1:5
scname <- run_ss3sim(iterations = iterations, simdf = df)

R version 4.0.5 (2021-03-31)

**Comment by kellijohnson-NOAA** [comment, marked as answer]
Thank you for your inquiry and sorry you are having problems with {ss3sim}. I am in the process of updating the vignette, which was brought to my attention in #394 where this same problem happened. Just remove the third line regarding `hess_always`. Please let me know if you run into any more problems and I will get the vignette updated right now.

**Reply by kinmic** [reply]
Ah brilliant, thank you very much for the quick response!
